src/lib/gundb/UserStorageClass.js

```diff
@@ -1706,7 +1749,7 @@ export class UserStorage {
             feedItem.otplStatus !== 'cancelled',
         )
         .map(feedItem => {
-          if (false == get(feedItem, 'data.receiptData', feedItem && feedItem.receiptReceived)) {
+          if (false === get(feedItem, 'data.receiptData', feedItem && feedItem.receiptReceived)) {
```

Comment:
needs to use ==
since it checks the existance and not boolean value
maybe use null == so its more clear

Reply:
in this case we could do simpler:

```suggestion
          if (!get(feedItem, 'data.receiptData', feedItem && feedItem.receiptReceived)) { 
```